dashboard/config/routes.rb

```diff
@@ -635,6 +635,7 @@
         collection do
           get 'get_feedback_from_teacher'
           get 'get_feedbacks'
+          get 'get_feedbacks_all'
```

Comment:
If we do go the API route, something like `GET api/v1/teacher_feedbacks/count` might be a nicer API for this particular purpose.

If we want to implement a "get all feedbacks" route, the Rails-y way of doing it would be to use the ["Resource Routing"](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default) `index` action `GET api/v1/teacher_feedbacks`.

```rb
# routes.rb
# Include :index in the 'only' list here:
resources :teacher_feedbacks, only: [:index, :create] do

# teacher_feedbacks_controller.rb
# And rename 'get_feedbacks_all' to 'index'
def index
```